zemljevid.js

```diff
@@ -33,15 +34,22 @@ var busIconKrsiDekret = L.icon({
 }); 
 
 var myIconlocation = L.icon({
-    iconUrl: 'lokacija.png',
+    iconUrl: 'location_accent-01.svg',
 	iconSize: [16, 16],
     iconAnchor: [8, 8],
     popupAnchor: [-3, -76]
 });
 
+var busDirection = L.icon({
+    iconUrl: 'bus_arrow.svg',
+	iconSize: [45, 45],
+    iconAnchor: [22.5, 37.5],
```

Comment:
Icon anchor should now be in the middle. Same thing in the bus icon.

(previously, the anchor was bottom-middle because there was that pinpoint-like shape)